packages/react-client/src/Client.ts

```diff
@@ -397,17 +385,6 @@ export class Client<PeerMetadata, TrackMetadata> extends (EventEmitter as {
       this.emit("authError", reason, this);
     });
 
-    this.tsClient.on("disconnected", () => {
```

Comment:
**thought:** i think it'd be good to just call state to snapshot instead of removing the whole handler